src/applications/ivc-champva/10-10D/config/submitTransformer.js

```diff
@@ -50,7 +50,7 @@ function transformApplicants(applicants) {
         app?.applicantOtherInsuranceCertification,
         app?.applicantHelplessCert,
       ],
-      address: app.applicantAddress ?? '',
+      address: app.applicantAddress ?? {},
```

Comment:
Unrelated, but an edge case error was discovered; having this be a blank object prevents an error when trying to set the `postal_code` key on this element later in the submit transformer.

Reply:
This edge case could only happen during testing, because the form requires that an applicant address be added; but it will save future headaches when unit-testing with incomplete form data.